beta-src/src/components/wd-popover/WDPopover.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import { useRef } from "react";
+import { Popover, Box } from "@mui/material";
+
+interface WDPopoverProps {
+  children: React.ReactNode;
+  handleClose: (event: React.MouseEvent<HTMLButtonElement>) => void; // A function that sets isOpen to false
```

Comment:
let's add the comments above each prop like so:
```
/**
 * A function that sets isOpen to false
 * /
handleClose: (event: React.MouseEvent<HTMLButtonElement>) => void; 
```

let's change the comment to:

A callback that is called when the Popover closes